ouroboros-network/src/Ouroboros/Network/Socket.hs

```diff
@@ -139,7 +139,7 @@ withNetworkNode
   => NetworkInterface ptcl Socket.AddrInfo IO r
   -> (NetworkNode Socket.AddrInfo IO r -> IO t)
   -> IO t
-withNetworkNode NetworkInterface {nodeAddress, protocols} k =
+withNetworkNode NetworkInterface {nodeAddress, nodeApplication} k =
```

Comment:
This only makes sense for `MuxApplications` that have a server side.

Reply:
This function was removed.